### DISCUSSION TITLE
Mint-X theme! And yes, it does pick up your highlight/accent color :)

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2388] [Show and tell]
Whew, THIS was a learning experience! I had to mess around a lot with trial-and-error and transparent .png textures to get this to work, but it _does_ work. 

This theme will pick up the values specified by `palette(highlight)` and `palette(highlighted-text)` where appropriate; I intend to ship this with a full collection of Mint-X-$COLOR palettes so just the one LXQt theme can accommodate the entire Mint-X spectrum. Here it is in vanilla (er...mint-chip?) flavor with default palette and matching Openbox theme.

![Mint-X-LXQt](https://user-images.githubusercontent.com/67122280/222988553-e3826d46-af4d-4bbf-9024-ca97ecadc632.png)

**Comment by jeroendiederen** [comment]
Do you have a package for this theme?

**Reply by AzumaHazuki** [reply]
I don't know what kind of package you mean, sorry...

...also, this theme has a problem: GTK theme engines use an operator called "shade" (and another called "mix") to produce the shaded menuitem highlights in Mint-X and also the glassy taskbutton highlights in themes like Shiki-Colors and the Murrine collection. I tried to fake it a bit here but if you're looking closely that green gradient is _not_ an exact match for what you'd get using the GTK theme.

And I have no idea, none, how to produce a semi-transparent .svg overlay that will make an exact match! There's no documentation I can find on how the shade function works internally, and even if there were, I don't know how to make gradients that work on luminance/perceptual lightness, only alpha ><;

**Comment by jeroendiederen** [comment]
I mean a .deb that installs everything.
